buildstockbatch/workflow_generator/residential/latest/__init__.py

```diff
@@ -0,0 +1,12 @@
+__version__ = "2024.07.19"
+version_info = {
+    "version": __version__,
+    "minimum_resstock_version": "3.2.0",
```

Comment:
Should update this version to 3.3.0 after the res-reporting branch updates the ResStock version in buildstock.rb to 3.3.0.